spec/basket_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "basket"
+
+describe Basket do
+
+  before (:each) do
+    @basket = Basket.new
```

Comment:
Awesome use of before block! I even prefer to use 'subject' for this a la http://www.betterspecs.org/#subject